### PR TITLE
Fix hanging socket test

### PR DESCRIPTION
**Pull request by lennym**
Async test wasn't calling `done`

### CODE REVIEW

File: test/services/websockets.test.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ describe('Web Socket Service', function () {
           }
         ];
         expect(store.getActions()).to.deep.equal(expectedActions);
+        done();
```
Comment (nelsonic) [inline test/services/websockets.test.js]: well spotted. 👍

Comment (lennym) [inline test/services/websockets.test.js]: The spot was @jackcarlisle's - I just `cherry-pick`-ed the commit as he's deep in other things.

**Comment by jruts** [comment]
@jackcarlisle is the bomb! :bomb:

**Comment by nelsonic** [comment]
@lennym will need to merge this into your `bugfix/date-formatting` branch to ensure that your PR passes: https://github.com/numo-labs/isearch-ui/pull/174

**Comment by lennym** [comment]
@nelsonic Already done!

**Comment by nelsonic** [comment]
sweet! 👍

**Pull request merged**